src/app.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// Write your code here
+import readline from 'readline';
+import { generateRandomNumber } from './modules/generateRandomNumber';
+import { checkIsValidUserInput } from './modules/checkIsValidUserInput';
+import { getBullsAndCows } from './modules/getBullsAndCows';
+
+const terminal = readline.createInterface(process.stdin, process.stdout);
+
+const numberToGuess = generateRandomNumber();
+
+terminal.question('Guess the number', (userNumber) => {
+  if (!checkIsValidUserInput(userNumber)) {
+    return;
```

Comment:
The program should provide feedback to the user if the input is invalid. Consider adding a message to inform the user that their input was not valid.